src/components/pokemon/pokemon.component.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { APIcatchPokemon, APIsetPokemonFree } from 'FetchAPI/fetch.methods';
+import { catchPokemon } from 'Redux/pokemons/pokemons.actions';
+import { setPokemonFree } from 'Redux/pokemons/pokemons.actions';
+
+import CustomButton from 'Components/custom-button/custom-button.component';
+
+import './pokemon.styles.scss';
+
+class Pokemon extends React.Component {
+  render() {
+    const {
+      id,
+      name,
+      status,
+      captureDate,
+      catchPokemon,
+      setPokemonFree
+    } = this.props;
+    return (
+      <div
+        className={`${
+          status !== 'free' ? 'Pokemon_captured' : 'Pokemon_captured-alt'

```

Comment:
move this sort of expressions from jsx. before return you can make variable like `pokemonMainContainerClasses` and just insert it in jsx like `<div className={pokemonMainContainerClasses}>`